apps/sms/index.html

```diff
@@ -63,7 +63,7 @@
     <link rel="stylesheet" type="text/css" href="shared/elements/gaia_sim_picker/style.css">
     -->
     <!-- Shared code -->
-    <script defer src="shared/js/l10n.js"></script>
+    <script type="application/javascript;version=1.8" defer src="shared/js/l20n.js"></script>
```

Comment:
`shared/js/l20n.js` is written in es6 and makes use of `const` and `let`.  Even though 1.8 has slightly different semantics for those keywords than es6 it doesn't actually hurt us at all. With `version-1.8` Gecko kindly agrees to run our code :)